### PR TITLE
deprecate safebreach v1

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/24912

## Description
Moved to DeprecatedContent pack

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/DeprecatedContent/ReleaseNotes/1_0_3.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+### Scripts
+- __SafeBreach__
```
Comment (ShahafBenYakir) [inline Packs/DeprecatedContent/ReleaseNotes/1_0_3.md:3]:
```suggestion
### Integrations
- __SafeBreach__
Deprecated - We recommend using SafeBreach v2 instead.

```

**Comment by ShahafBenYakir** [comment]
@guyfreund Please remove from `conf.json` too

File: Packs/DeprecatedContent/Integrations/integration-SafeBreach.yml [hunk 2]
```diff
@@ -197,3 +197,5 @@ script:
       type: number
     description: Fetch the breach simulation results
   runonce: false
+tests:
+  - No tests
```
Comment (ShahafBenYakir) [inline Packs/DeprecatedContent/Integrations/integration-SafeBreach.yml:201]:
```suggestion
  - No tests: deprecated. Test was moved to NonCircleTests folder.
```

**Comment by kirbles19** [comment]
@guyfreund Done

**Comment by Itay4** [comment]
lets merge from master here

**Pull request merged**